### PR TITLE
Publish the Gradle plugin from CI on release jobs on master

### PR DESCRIPTION
**Pull request by oldergod**
Thanks to @martinbonnin for the help.

I didn't put the time to know how to version check inside a github if statement, so doing it in kts instead.

### CODE REVIEW

**Review by swankjesse — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -180,7 +180,9 @@ jobs:
         run: sudo xcode-select -switch "/Applications/Xcode_12.app"
 
       - name: Upload Artifacts
-        run: ./gradlew -p wire-library clean publish --stacktrace --no-parallel
+        run: |
+          ./gradlew -p wire-library clean publish --stacktrace --no-parallel
+          ./gradlew -p wire-library publishPluginToGradlePortalIfRelease --stacktrace -Pgradle.publish.key="${{ secrets.GRADLE_PUBLISH_KEY }}" -Pgradle.publish.secret="${{ secrets.GRADLE_PUBLISH_SECRET }}"
```
[the remark this answers is not in the document]
Comment (oldergod) [reply .github/workflows/build.yml:185]: I didn't find any trick to set `gradle.publish.key` in any other way

[the remark this answers is not in the document]
Comment (oldergod) [reply .github/workflows/build.yml]: Ok, I've tried something else. Let me know.

[the remark this answers is not in the document]
Comment (martinbonnin) [reply .github/workflows/build.yml]: 🤞 🤞

**Review by swankjesse — APPROVED** [review]

**Comment by oldergod** [comment]
:sad:

> Caused by: java.lang.IllegalArgumentException: Missing publishing keys. Please set gradle.publish.key/gradle.publish.secret system properties or GRADLE_PUBLISH_KEY/GRADLE_PUBLISH_SECRET env variables or login using the login task.

**Pull request merged**